sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/GenericArrayDataBenchmark.scala

```diff
@@ -58,7 +58,7 @@ object GenericArrayDataBenchmark extends BenchmarkBase {
     }
 
     benchmark.addCase("arrayOfAnyAsSeq") { _ =>
-      val arr: Seq[Any] = new Array[Any](arraySize)
+      val arr: scala.collection.Seq[Any] = new Array[Any](arraySize)
```

Comment:
`Seq` in Scala 2.12 representative `scala.collection.Seq`, but in  in Scala 2.12 representative `scala.collection.immutable.Seq`, for the sake of fairness, change to  declared as `scala.collection.Seq` here